### ISSUE TITLE
Show missing parent state

### ISSUE THREAD

**Issue by MindFreeze** [MindFreeze/ha-sankey-chart#15]
Spin off from #2 

Similar to remaining/unaccounted state in children, it would be useful to see unaccounted parent state.

**Comment by Madelena** [comment]
Yep! That's what my original request is for. 👍😄 Looking forward to it.
